plenum/test/replica/test_consensus_data_helper.py

```diff
@@ -38,8 +38,8 @@ def test_pp_storages_ordering(pre_prepare, consensus_data_helper: ConsensusDataH
 
 
 def test_pp_storages_freeing(pre_prepare, consensus_data_helper: ConsensusDataHelper):
-    consensus_data_helper.consensus_data.prepared.append(pre_prepare)
-    consensus_data_helper.consensus_data.preprepared.append(pre_prepare)
+    consensus_data_helper.consensus_data.prepared.append(batch_id(pre_prepare))
+    consensus_data_helper.consensus_data.preprepared.append(batch_id(pre_prepare))
```

Comment:
Do we actually need to use PrePrepares in this test instead of BatchIDs?

Reply:
`ConsensusDataHelper` deals with PrePrepares as input parameters, so I think it's OK to keep it like this.